src/commcare_cloud/commands/terraform/constants.py

```diff
@@ -17,6 +19,7 @@
 ^/a/([\w\.:-]+)/cloudcare/apps/report_formplayer_error
 ^/a/([\w\.:-]+)/data/export/custom/download_data_files/$
 ^/a/([\w\.:-]+)/data_dictionary/import$
+^/a/([\w\.:-]+)/dhis2/map/(\w+)/$
```

Comment:
My own change

Reply:
I approve this change. If you would like to merge this before any open questions about other changes are resolved, you could PR just this line, and leave other changes for later.